BuildNotifications.Core/Pipeline/Tree/Search/Criteria/BaseStringCriteria.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BuildNotifications.Core.Config;
+using BuildNotifications.PluginInterfaces.Builds;
+
+namespace BuildNotifications.Core.Pipeline.Tree.Search.Criteria
+{
+    internal abstract class BaseStringCriteria : BaseSearchCriteria
```

Comment:
For what exactly? For checking whether the current search pattern is already equal to the input?
Whats the benefit of the string comparer?

Reply:
Okay I have no idea why what past-me was suggesting this. The StringComparers are useless.
However I would suggest using CurrentCulture for comparision since that is how the user expects their string comparision to work like. 

I remember a crash that was caused by this "feature" of the danish language.
https://stackoverflow.com/questions/6535910/string-startswith-issue-with-danish-text

Reply:
Done